### PR TITLE
Updates To Conference Dates, logo, registration links, and other edits

### PR DESCRIPTION
**Pull request by kipkurui**

### CODE REVIEW

**Comment by bkmgit** [comment]
Thanks. Is it possible to also add SVG/EPS/PDF files for the logo? Vector formats display better for designs that are not very complicated.

**Comment by kipkurui** [comment]
I have added an SVG format

> Thanks. Is it possible to also add SVG/EPS/PDF files for the logo? Vector formats display better for designs that are not very complicated.

**Comment by bkmgit** [comment]
May also be helpful to indicate virtual.

**Comment by bkmgit** [comment]
@Fnyasimi @karegapauline reviews would be helpful

**Comment by kipkurui** [comment]
> 

Yes, we need to capture that on the website. or did you mean the logo?

**Comment by bkmgit** [comment]
Added a suggestion to the home page. Maybe the section "It is a learning and networking event..." could also be changed to "It is a virtual learning and networking event...", but other suggestions are welcome.  We want anyone who registers to know that travel is not required.

**Review by karegapauline — COMMENTED** [review]
Changes are good to merge

**Comment by bkmgit** [comment]
@karegapauline Have left comments. They do not all need to be implemented. Maybe registration can start on 10/11 March?

File: pages/en/codeofconduct.markdown [hunk 1]
```diff
@@ -4,3 +4,33 @@ title: Code of Conduct
...
+
+## Unaccepted Behavior (Don’ts)
+
+- Verbal or written comments that exclude people on the basis of membership to either community (BHKi or OpenScienceKE) or a specific group.
+- Causing members discomfort or fear for their safety especially through stalking, intimidation, harassment or cyber bullying.
+- Using harsh or sexual language, racist or inappropriate jokes.
+- Sexual or violent display of images or any media content in any event.
+
+## Enforcement
+To avoid any situation that breaches our Code of Conduct, we advise our members to not post anything that doesn’t pertain to the events on our online platforms (Slack) or community forums without any approval from the organising committee.
```
[the remark this answers is not in the document]
Comment (kipkurui) [reply pages/en/codeofconduct.markdown:32]: Sounds better, thanks!

File: pages/en/register.markdown [hunk 2]
```diff
@@ -4,3 +4,13 @@ title: Register
...
 lang: en
 ---
+
+## Jisajili Mkutano
+
+[Jisajili hapa](https://www.eventbrite.com/e/bioinformatics-and-open-science-skills-conference-2022-tickets-289432679937) kuhudhuria mkutano wa BOSS Con 2022. 
+ 
+ Mwanzo wa kujisajili ni tarehe 7 Machi 2022. 
+
+## Kutuma muhtasari:
```
[the remark this answers is not in the document]
Comment (kipkurui) [reply pages/en/register.markdown:14]: Swahili is hard. Thanks for the changes

File: pages/en/index.markdown [hunk 3]
```diff
@@ -18,11 +18,11 @@ interact with other researchers and more.
 
 - Conference sessions: 26-29 April 2022
```
[the remark this answers is not in the document]
Comment (kipkurui) [reply pages/en/index.markdown:19]: Yes, this is something we've overlooked. We sure ned to make that Very Clear on all sites

**Comment by karegapauline** [comment]
@bkmgit . The tickets open today but we can announce it officially on 10th, that's okay. We should have everything updated by then. I've seen you've made changes to the wording on the homepage and the CoC. @Fnyasimi should be able to make changes to the banner. 

Is anything else missing or pending, before we merge?

**Comment by bkmgit** [comment]
@karegapauline Would like to examine a couple other registration options, so if can wait until 10, that would be great.

**Comment by kipkurui** [comment]
My suggestion is that we should not spend too much time on this. What are the significant downsides of Eventbrite? We could even use the Zoom registration. It is not a big deal since the event is free; we need an option to know who is keen to attend. Call for abstracts, and the submission system will be done via OpenReview. 

@bkmgit, is the registration link what is delaying this merge?

**Comment by bkmgit** [comment]
There are probably better options for registration, so would like to investigate and discuss those if possible.

**Comment by kipkurui** [comment]
@bkmgit removed the link for now.

**Comment by bkmgit** [comment]
@Fnyasimi If ok for you, please merge.

File: pages/en/index.markdown [hunk 4]
```diff
@@ -16,13 +16,13 @@ interact with other researchers and more.
...
+- Online conference sessions: 26-29 April 2022
 
-- Abstract submission deadlines:
+- Abstract submission deadlines: 4th April 2022
 
-- Abstract notification:
+- Abstract notification: 18th April 2022
 
-- Registration open:
+- Registration open: 7th March 2022
```
Comment (bkmgit) [inline pages/en/index.markdown:25]:
```suggestion
- Registration open: 10th March 2022
```

File: pages/en/register.markdown [hunk 5]
```diff
@@ -4,3 +4,13 @@ title: Register
 permalink: /register/
 lang: en
 ---
+
+## Meeting Registration
+
+<!--- [Register](https://www.eventbrite.com/e/bioinformatics-and-open-science-skills-conference-2022-tickets-289432679937) to attend BOSS Con 2022. --->
+ 
+ Registration opens on 7 March 2022.
```
Comment (bkmgit) [inline pages/en/register.markdown:12]:
```suggestion
 Registration opens on 10th March 2022. 
```

File: pages/sw/index.markdown [hunk 6]
```diff
@@ -14,13 +14,13 @@ Mkutano wa BOSS 2022 ni tukio wa tano na wa mwisho za BOSS kutoka Oktoba 2021 ha
...
+- Mkutano kwa mtandao: 26-29 Aprili 2022
 
-- Mwisho wa kutuma muhtasari:
+- Mwisho wa kutuma muhtasari: 4 Aprili 2022
 
-- Kauli ya Wasemaji:
+- Kauli ya Wasemaji: 18 Aprili 2022
 
-- Aanzisho wa Usajili:
+- Aanzisho wa Usajili: 7 Machi 2022
```
Comment (bkmgit) [inline pages/sw/index.markdown:23]:
```suggestion
- Aanzisho wa Usajili: 10 Machi 2022
```

File: pages/en/register.markdown [hunk 7]
```diff
@@ -4,3 +4,13 @@ title: Register
...
+
+## Meeting Registration
+
+<!--- [Register](https://www.eventbrite.com/e/bioinformatics-and-open-science-skills-conference-2022-tickets-289432679937) to attend BOSS Con 2022. --->
+ 
+ Registration opens on 7 March 2022. 
+
+## Abstract Submissions:
+
+<Add a call for abstract submissions>
```
Comment (Fnyasimi) [inline pages/en/register.markdown:16]: @kipkurui  you can provide this later on. In a different pull

**Review by Fnyasimi — APPROVED** [review]

**Pull request merged**